ios/FluentUI/Command Bar/CommandBarItem.swift

```diff
@@ -108,6 +108,12 @@ open class CommandBarItem: NSObject {
     /// Set `isSelected` to desired value in this handler. Default implementation is toggling `isSelected` property.
     @objc public var itemTappedHandler: ItemTappedHandler
 
+    /// If set, the `UIView` returned from the closure will be shown inside `CommandBarButton` instead of the `title` and `image`
+    @objc public var customControlView: (() -> UIView)?
```

Comment:
how do we handle ipad pointer interaction on this ?

Reply:
Currently the `CommandBarButton` does not support any additional pointer interaction other than the default on-press highlighting. Since this is a custom view, I don't think there should be any default behavior provided by our the core component. Any pointer interactions should be defined by the view provider.